### ISSUE TITLE
Disable Mesa hack for 21.x ?

### ISSUE THREAD

**Issue by simonschmeisser** [ros-visualization/rviz#1701]
In #1508 you, @rhaschke , stated that in mesa 21.0.3 the bug requiring the old GLSL version has been fixed. Yet here https://github.com/rhaschke/rviz/blob/62ba5cd26344dceeda1b66ada5dd0740f218cbf3/src/rviz/ogre_helpers/render_system.cpp#L193 it still checks for Mesa 2x.y (including Mesa 21.y)

Should that check be limited?

I have read (but not yet confirmed) that Octomap rendering is significantly faster with the newer shaders (but I have seen slowdowns compared to older rviz versions)

**Comment by rhaschke** [comment]
Yes, I think this workaround can be limited to Mesa < 21.0.3. Could you test this and file a PR?
